code/datums/diseases/advance/symptoms/sneeze.dm

```diff
@@ -19,8 +19,8 @@ Bonus
 /datum/symptom/sneeze
 
 	name = "Sneezing"
-	stealth = -2
-	resistance = 3
+	stealth = -1

```

Comment:
@Gunhog unless I'm mistaken, I'm pretty sure this is actually decreasing the stealth penalty that the symptom gives, not increasing the penalty. It's a buff. Positive values improve stealth/transmission/resistance/etc.. while negative values decrease it.